website/pages/partials/from-1.5/builds/example-block.mdx

```diff
@@ -0,0 +1,27 @@
+```hcl
+# build.pkr.hcl
+build {
+
+    sources = [
+        # use the plural `sources` block to simply use sources
+        # without changing any field.
+        "source.amazon-ebs.example",
+    ]
+
+    source "source.amazon-ebs.example" {
```

Comment:
I think this should just work; the logs will be confusing though 🤔

Reply:
Yes:

```hcl
source "null" "test" {
    communicator = "none"
}

source "null" "potato" {
    communicator = "none"
}

build {
    sources = [
        "sources.null.test",
        "sources.null.test",
        "sources.null.potato",
    ]
}
```

```shellsession
> go run . build ./.vscode/hcl2/null/does_nothing
null.test: output will be in this color.
null.test: output will be in this color.
null.potato: output will be in this color.

Build 'null.test' finished.
Build 'null.test' finished.
Build 'null.potato' finished.

==> Builds finished. The artifacts of successful builds are:
--> null.test: Did not export anything. This is the null builder
--> null.potato: Did not export anything. This is the null builder
```

Reply:
What do you think about this team ? May be we could force users of a singular `source` block to name them, for example:

```hcl
build {
    source "sources.null.potato" "specific-name" {
        
    }
}
```

Another thing to do would be to tell people to avoid doing that in the docs but I feel like this won't solve anything.